### PR TITLE
Refactor styling for article comment counts

### PR DESCRIPTION
**Pull request by phamann**
Fix's the missing comment count icon within the article meta data.
### Before

google chrome
### After

google chrome

/cc @sndrs @mattosborn

### CODE REVIEW

File: static/src/stylesheets/module/content/_content.global.scss [hunk 1]
```diff
@@ -61,13 +61,20 @@
...
-        }
-        .i {
+
+        .i, .inline-icon {
             position: absolute;
             left: 0;
         }
+
+        @include mq(leftCol) {
+            padding-left: 15px;
```
Comment (sndrs) [inline static/src/stylesheets/module/content/_content.global.scss]: is this exempt from `$gutter`?

Comment (OliverJAsh) [inline static/src/stylesheets/module/content/_content.global.scss]: ^

**Comment by OliverJAsh** [comment]
Otherwise :+1:

File: static/src/stylesheets/module/content/_content.global.scss (hunk 1 above)
Comment (phamann) [inline static/src/stylesheets/module/content/_content.global.scss]: Nope you are right. Just that I didn't write it, only moved it :smile: but I will refactor now.

**Pull request merged**